backend/app/engine/epochs_settings.py

```diff
@@ -68,6 +70,7 @@ def register_epoch_settings():
             matched_rewards=PreliminaryMatchedRewards(),
             ppf=NotSupportedPPFCalculator(),
             community_fund=NotSupportedCFCalculator(),
+            leftover=DefaultLeftover(),
```

Comment:
we named our class as ```DefaultLeftover``` but it's not used as default one anymore (```LeftoverWithPPF``` is), didn't we? 

maybe let's stick to the convention ```PreliminaryLeftover``` instead of ```DefaultLeftover``` since "default is not a default one" for next epochs anymore?

Reply:
ok